person.rb

```diff
@@ -31,6 +32,11 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def rent_book(book)
+    @books_rented << book
+    book.rent_book = self
+  end
```

Comment:
- [ ] the method for achieving the has-many relationship should look a little different, please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more details.